docs/src/list_of_supported_models.md

```diff
@@ -1,5 +1,8 @@
 # [List of Supported Models](@id model_list)
 
+For a list of models organized around function ("classification", "regression", etc.), see
+the [Model Browser](@ref).
```

Comment:
Just to make sure, the model browser will now automatically include a "Correcting for Class Imbalance" section with models from "Imbalance.jl", right?

Reply:
Yes. Here it is on my local preview:

<img width="531" alt="Screen Shot 2023-10-11 at 9 52 34 AM" src="https://github.com/alan-turing-institute/MLJ.jl/assets/30517088/46fd30c1-0548-4eb1-85ec-502db2b10d2b">